### DISCUSSION TITLE
Multiway equality is not working ?

### DISCUSSION THREAD

**Discussion by ksubileau** [helmfile/helmfile#1072] [Questions] [answered]
Hi,

I tried to write something like this in a values.yaml.gotmpl file :
```
key: {{ env "MY_VARIABLE" | eq "apple" "banana" "strawberries" }}
```
Based on the [template package documentation](https://pkg.go.dev/text/template@go1.21.3#hdr-Functions), I expected this to return true if `MY_VARIABLE` is equal to any value among `apple`, `banana` or `strawberries`.

> For simpler multi-way equality tests, eq (only) accepts two or more arguments and compares the second and subsequent to the first returning in effect `arg1==arg2 || arg1==arg3 || arg1==arg4 ...`

But that doesn't seem to be the case, true is only returned if the value is equal to the first argument (`apple` in my example), ignoring subsequent parameters.

Why doesn't this follow what the documentation says? Would it be possible to add this functionality?

**Comment by yxxhero** [comment, marked as answer]
```
{{ eq (env "MY_VARIABLE") "apple" "banana" "strawberries" }}
```

**Reply by yxxhero** [reply]
@ksubileau please try as I advice.

**Reply by ksubileau** [reply]
Oh great, it works, thank you! But we must recognize that the difference is rather subtle...

**Reply by yxxhero** [reply]
@ksubileau it's different totally.
